ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -727,6 +755,8 @@ encodeShelleyQuery query = case query of
       CBOR.encodeListLen 2 <> CBOR.encodeWord8 26 <> LC.toEraCBOR @era dreps
     GetCommitteeMembersState coldCreds hotCreds statuses ->
       CBOR.encodeListLen 4 <> CBOR.encodeWord8 27 <> toCBOR coldCreds <> toCBOR hotCreds <> LC.toEraCBOR @era statuses
+    GetFilteredVoteDelegatees stakeCreds ->
+      CBOR.encodeListLen 2 <> CBOR.encodeWord8 28 <> LC.toEraCBOR @era stakeCreds
```

Comment:
We need a corresponding deserialization case in `decodeShelleyQuery`.

Reply:
Oh good point, I overlooked this, sorry :(